ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/ImagesAtMostRisk.tsx

```diff
@@ -0,0 +1,183 @@
+        sort: [{ id: 'Image Risk Priority', desc: 'false' }],
+    });
+    return `${vulnManagementImagesPath}${queryString}`;
+}
+
+export const imagesQuery = gql`
+    query getImages($query: String) {
+        images(
+            query: $query
+            pagination: { limit: 6, sortOption: { field: "Image Risk Priority", reversed: false } }
+        ) {
+            id
+            name {
+                remote
+                fullName
+            }
+            priority
+            vulnCounter {
+                important {
+                    total
+                    fixable
+                }
+                critical {
+                    total
+                    fixable
+                }
+            }
+        }
+    }
+`;
+
+function getQueryVariables(searchFilter: SearchFilter, statusOption: ImageStatusOption) {
+    const query =
+        statusOption === 'Active' && !isResourceScoped(searchFilter)
+            ? 'Namespace:*'
+            : getRequestQueryStringForSearchFilter(searchFilter);
+    return { query };
+}
+
+const fieldIdPrefix = 'severe-cve-images';
+
+type ImageStatusOption = 'Active' | 'All';
+
+function ImagesAtMostRisk() {
+    const { isOpen: isOptionsOpen, onToggle: toggleOptionsOpen } = useSelectToggle();
+    const { searchFilter } = useURLSearch();
+
+    const [cveStatusOption, setCveStatusOption] = useState<CveStatusOption>('Fixable');
+    const [imageStatusOption, setImageStatusOption] = useState<ImageStatusOption>('All');
+
+    const variables = getQueryVariables(searchFilter, imageStatusOption);
+    const { data, previousData, loading, error } = useQuery<ImageData>(imagesQuery, {
+        variables,
+    });
+
+    const imageData = data || previousData;
+    const isScopeApplied = isResourceScoped(searchFilter);
+
+    return (
+        <WidgetCard
+            isLoading={loading || !imageData}
+            error={error}
+            header={
+                <Flex direction={{ default: 'row' }}>
+                    <FlexItem grow={{ default: 'grow' }}>
+                        <Title headingLevel="h2">{getTitle(searchFilter, imageStatusOption)}</Title>
+                    </FlexItem>
+                    <FlexItem>
+                        <Dropdown
+                            className="pf-u-mr-sm"
+                            toggle={
+                                <DropdownToggle
+                                    id={`${fieldIdPrefix}-options-toggle`}
+                                    toggleVariant="secondary"
+                                    onToggle={toggleOptionsOpen}
+                                >
+                                    Options
+                                </DropdownToggle>
+                            }
+                            position="right"
+                            isOpen={isOptionsOpen}
+                        >
+                            <Form className="pf-u-px-md pf-u-py-sm" style={{ minWidth: '250px' }}>
```

Comment:
The `250px` is an arbitrary minimum width the prevents the text in the option buttons from wrapping to a second line.